Retry until ng-select is ready to accept events in `custom_actions_spec.rb`

aaron-contreras opened the pull request:
It seems that  ng-select is not ready to receive events for some reason. I'm unaware of a way to expect it to be ready to receive events other attempting to set the value on the conditions multiple times.

Because the problem doesn't seem to be in finding the dropdown item to click on in the autocompleter but in the expectation for the the value to ACTUALLY be set, retrying on this sequence seems the way to go.

I did a bulk run while under 96% CPU usage with all passing.

This seems to be it! 🍾


```shell
-> % RSPEC_RETRY_COUNT=2 script/bulk_run_rspec ./spec/features/work_packages/custom_actions/custom_actions_spec.rb:137
1 tests to run
================================================================================
Running ./spec/features/work_packages/custom_actions/custom_actions_spec.rb:137 5 times
================================================================================
------------------------------------ Run 0 -------------------------------------
ok
------------------------------------ Run 1 -------------------------------------
ok
------------------------------------ Run 2 -------------------------------------
ok
------------------------------------ Run 3 -------------------------------------
ok
------------------------------------ Run 4 -------------------------------------
ok
PASSED
```

aaron-contreras committed:
Fix: Retry until ng-select is ready to accept events

It seems that  ng-select is not ready to receive events for some
reason. I'm unaware of a way to expect it to be ready to receive events
other attempting to set the value on the conditions multiple times.

Because the problem doesn't seem to be in finding the dropdown item to
click on in the autocompleter but in the expectation for the the value
to ACTUALLY be set, retrying on this sequence seems the way to go.

I did a bulk run while under 96% CPU usage with all passing.

This seems to be it! 🍾

cbliard reviewed (approved):
Thanks! Let's see how it goes :crossed_fingers:

aaron-contreras committed:
Wait for assignee activity to be present before escalation

Waiting for the assignee activity to reflect and then for the
escalation mention to populate is much less flaky as there shouldn't
be any debouncing collissions if the spec is running too fast and under
heavy CPU load
